docs/guide/core-concepts.md

```diff
@@ -1,28 +1,188 @@
-Tile Layers
+Geographical Information Systems (GIS), like any specialized field, has a wealth
+of jargon and unique concepts. When represented in software, these concepts
+can sometimes be skewed or expanded from their original forms. Three of
+these critical to effective use of Geotrellis are *Projections*, *Extents*,
+and *Layout Definitions*. We give a thorough definition of many of the core concepts
+here, while referencing the Geotrellis objects and source files backing them.
+
+This document aims to be informative to new and experienced GIS users alike.
+If GIS is brand, brand new to you, [this document](https://www.gislounge.com/what-is-gis/) is
+ a useful high level overview.
+
+# Raster Data
```

Comment:
Here and elsewhere for this file in particular, let's use a single `#`. The underline can be dropped.